container/lxd/initialisation_test.go

```diff
@@ -529,3 +535,104 @@ func (s *InitialiserSuite) TestBridgeConfigurationWithNewSubnet(c *gc.C) {
 	actualValues := parseLXDBridgeConfigValues(result)
 	c.Assert(actualValues, gc.DeepEquals, expectedValues)
 }
+
+type ConfigureInitialiserSuite struct {
+	initialiserTestSuite
+	testing.PatchExecHelper
+}
+
+var _ = gc.Suite(&ConfigureInitialiserSuite{})
+
+func (s *ConfigureInitialiserSuite) SetUpTest(c *gc.C) {
+	s.BaseSuite.SetUpTest(c)
+	// Fake the lxc executable for all the tests.
+	testing.PatchExecutableAsEchoArgs(c, s, "lxc")
+	testing.PatchExecutableAsEchoArgs(c, s, "lxd")
+}
+
+func (s *ConfigureInitialiserSuite) TestConfigureLXDBridge(c *gc.C) {
+	ctrl := gomock.NewController(c)
+	defer ctrl.Finish()
+	cSvr := lxdtesting.NewMockContainerServer(ctrl)
+	s.PatchForProxyUpdate(c, cSvr, true)
+
+	profile := &api.Profile{
+		ProfilePut: api.ProfilePut{
+			Devices: map[string]map[string]string{
+				"eth0": {
```

Comment:
A comment here too - the presence of this bridged NIC means the profile is no updated with a newly created one.